static/src/javascripts/projects/common/modules/identity/ad-prefs/wordings.js

```diff
@@ -7,30 +8,62 @@ type CheckboxWording = {
     text?: string,
 };
 
+type QuestionWording = {
+    title: string,
+    text?: Node,
```

Comment:
```suggestion
    text: Node,
```
No need for this to be optional I think?

Reply:
ah nevermind!